src/main/java/cultfit/Error.java

```diff
@@ -0,0 +1,19 @@
+package cultfit;
+
+public class Error {
+    private String msg;
+    private boolean isError;
+
+    public Error(String msg, boolean isError) {
+        this.msg = msg;
+        this.isError = isError;
+    }
```

Comment:
Better to have class name something like Response rather than Error

Reply:
agreed